### PR TITLE
removed tenant from RoleEnum ran db migrations as per Contrib doc all…

### LINKED ISSUE #432: Remove the tenant role
**Purpose**
Remove the UserRole Tenant - Tenant is not a role and should be removed.

**Proposed Solution**
Remove tenant from the RoleEnum in the models/user. 

**Acceptance Criteria**

- [ ] all tests passing after removal of tenant role.

### PR DESCRIPTION
**Pull request by whiletrace**
… tests passing

### What issue is this solving?

Closes codeforpdx/dwellingly-app/issues/432

### Any helpful knowledge/context for the reviewer?
deleted the tenant attribute from the class RoleEnum 
ran migration as per the Contrib Docs all tests pass as is 
reseeding may be necessary since db table structure has changed  
Any new dependencies to install?
Any special requirements to test?

### Please make sure you've attempted to meet the following coding standards
- [ x] Code builds successfully
- [x ] Code has been tested and does not produce errors
- [x ] Code is readable and formatted
- [ x] There isn't any unnecessary commented-out code

### CODE REVIEW

File: migrations/versions/f7cb1b951d51_message.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
+"""message
```
Comment (NickSchimek) [inline migrations/versions/f7cb1b951d51_message.py]: Can we remove this file? We're currently not using alembic and shouldn't be running migrations. I think there is a note in the README, but if it is not clear I'd be happy to get a PR for it.

**Review by NickSchimek — CHANGES_REQUESTED** [review]
Let's remove the migration file. We're not using alembic yet. We will once we get closer to deployment. For now we just drop and rebuild the db with `pipenv run flask db recreate`

File: migrations/versions/f7cb1b951d51_message.py (hunk 1 above)
Comment (whiletrace) [reply migrations/versions/f7cb1b951d51_message.py]: ok Ill post a quick fix, The contrib docs are a bit confusing on this matter, maybe a little format or punctuation there could help. but Ill get rid of the migration file.

**Review by NickSchimek — APPROVED** [review]
Thanks @whiletrace

**Pull request merged**